executeUserScript.py

```diff
@@ -1,47 +1,22 @@
-import sys
-import argparse
 import traceback
 
 
-class executeUserScript():
-    def __init__(self):
-        print("***Execute User Script***\n")
-
-    def parsingCommandLineArguments(self):
-        ''''
-            capture command line arguments.
-            :return
-                dictionary
-        '''
-        parser = argparse.ArgumentParser()
-        parser.add_argument('-S', '--userscript', help='User script Required!', required=True)
-        arguments = parser.parse_args()
-        return vars(arguments)
+class executeUserScript(object):
 
     def executeScript(self, scriptName):
```

Comment:
Are we only allowing a script to be used to run our program? Can they not provide command-line arguments to call and start their program?

Reply:
is this discussed already? do you want to bring it up to the next meeting? Up to now this was the agreement. This should not stop this pull request.

Reply:
Not blocking, will need to get clarification. But could you please rename the file to match the rest of the project?

Reply:
Naming of the files have been changed. Please let me know if you have any question. 

Thanks!